_case_studies/11_targeting_variants_for_maximum_impact.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Targeting variants for maximum impact using the CausalTune library
```

Comment:
"Targeting variants" sounds very technical.  Is there a way to re-word the title to make it clearer who would want to read this case study and why? In the tutorial itself, several of the phrases used in sections seem to be clearer --> "improving key business metrics", "optimizing clickthrough rates", etc.  Would one of those work for the tutorial title?

Reply:
Thank you, changed to "Improving business metrics for maximum impact using the CausalTune library"